packages/plugins/rubygem-elastic-apm/rubygem-elastic-apm.spec

```
@@ -0,0 +1,100 @@
+# Generated from elastic-apm-2.6.1.gem by gem2rpm -*- rpm-spec -*-
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name elastic-apm
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 2.6.1
+Release: 1%{?dist}
+Summary: The official Elastic APM agent for Ruby
+Group: Development/Languages
+License: Apache-2.0
+URL: https://github.com/elastic/apm-agent-ruby
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start generated dependencies
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby >= 2.3.0
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix_ror}rubygem(concurrent-ruby) >= 1.0
+Requires: %{?scl_prefix_ror}rubygem(concurrent-ruby) < 2
+Requires: %{?scl_prefix}rubygem(http) >= 3.0
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}ruby >= 2.3.0
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+# end generated dependencies
+
+%description
+The official Elastic APM agent for Ruby.
+
+
+%package doc
+Summary: Documentation for %{pkg_name}
+Group: Documentation
+Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{pkg_name}.
+
+%prep
+%{?scl:scl enable %{scl} - << \EOF}
+gem unpack %{SOURCE0}
+%{?scl:EOF}
+
+%setup -q -D -T -n  %{gem_name}-%{version}
+
+%{?scl:scl enable %{scl} - << \EOF}
+gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
+%{?scl:EOF}
+
+%build
+# Create the gem as gem install only works on a gem file
+%{?scl:scl enable %{scl} - << \EOF}
+gem build %{gem_name}.gemspec
+%{?scl:EOF}
+
+# %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
+# by default, so that we can move it into the buildroot in %%install
+%{?scl:scl enable %{scl} - << \EOF}
+%gem_install
+%{?scl:EOF}
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -pa .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+%files
+%dir %{gem_instdir}
+%exclude %{gem_instdir}/.gitignore
+%{gem_instdir}/.hound.yml
+%exclude %{gem_instdir}/.rubocop.yml
+%{gem_instdir}/CODE_OF_CONDUCT.md
```

Maybe this should be part of `%files doc`?

```suggestion
%doc %{gem_instdir}/CODE_OF_CONDUCT.md
```